packages/checkout-ui-extensions/src/extension-points/api/standard/index.ts

```diff
@@ -428,7 +428,7 @@ export interface Address {
   address2?: string;
   city?: string;
   postalCode?: string;
-  countryCode: CountryCode;
+  countryCode?: CountryCode;
```

Comment:
This type change would be an enhancement, not a breaking change as it is additive in nature.